.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     - name: Build RPMs
       run: ./automation/rpm.sh
     - name: Upload artifacts
-      uses: actions/upload-artifact@v2
+      uses: ovirt/upload-rpms-action@v1
       with:
-        name: artifacts
-        path: ${{ env.EXPORT_DIR }}
+        directory: ${{ env.EXPORT_DIR }}
+        distro: el8stream
```

Comment:
In the current form this action is not very useful. It seem that the only
service it gives us is naming the artifact "artifact-el8stream".

Reply:
It makes possible to apply contingent future changes globally across oVirt, which is useful by itself.

Reply:
Considering what was discussed on sync I'll wait with this change for the distro auto detection.